packages/functional-tests/tests/oauth/oauthPermissions.spec.ts

```diff
@@ -2,9 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { use } from 'passport';
 import { test, expect } from '../../lib/fixtures/standard';
 
-let email;
+//let email;
 const password = 'passwordzxcv';
```

Comment:
Could this value be a constant in standard.ts that is re-used and imported? I think this would reduce brittleness, since this value is strongly coupled with an email.